docs/pages/configuration/variables/basics.mdx

```diff
@@ -77,6 +78,20 @@ vars:
   password: true
 ```
 
+</TabItem>
+
+<TabItem value="NoCache">
+
+```yaml {3,5-6}
```

Comment:
This highlights multiple lines within the snippet, so yaml {3,5-7} is needed here